desktop/src/app.rs

```diff
@@ -100,10 +101,10 @@ impl App {
 
         // Poll UI events.
         let event_loop = self.event_loop.take().expect("App already running");
-        event_loop.run(move |event, _window_target, control_flow| {
+        event_loop.run(move |event, elwt| {
```

Comment:
Huh, it took me a while to figure out that `elwt` is for `EventLoopWindowTarget`.
It can stay like this (this is a common pattern after all, just for an unfamiliar - to me - struct name), or it can be perhaps `target`? Even I'm not sure which one would be better...

Reply:
Yeah it felt funny to me too, I got it from the winit docs and felt I should be consistent with what they use 😅

Reply:
Okay, if they use it, maybe it's better that we do too, yeah.